content/maintainers-guide/project-organization.md

```diff
@@ -0,0 +1,85 @@
+- Marion François ([@marionf](https://github.com/marionf))
+- Mateus Shirlaw ([@MatShir](https://github.com/MatShir))
+
+### Maintainer
+
+Maintainers provide the project's technical vision, review code contributions, and ensure that the software develops in line with the product vision.
+
+Current maintainers:
+
+- Franck Lefèvre ([@Progi1984](https://github.com/Progi1984))
+- Ibrahima Sow ([@sowbiba](https://github.com/sowbiba))
+- Jonathan Lelièvre ([@jolelievre](https://github.com/jolelievre))
+- Krystian Podemski ([@kpodemski](https://github.com/kpodemski))
+- Mathieu Ferment ([@matks](https://github.com/matks))
+- Matthieu Rolland ([@matthieu-rolland](https://github.com/matthieu-rolland))
+- Pablo Borowicz ([@eternoendless](https://github.com/eternoendless))
+- Pierre Rambaud ([@PierreRambaud](https://github.com/PierreRambaud))
+- Pululu Kinanga André ([@PululuK](https://github.com/PululuK))
+- Thomas Baccelli ([@atomiix](https://github.com/atomiix))
+- Valentin Szczupak ([@NeOMakinG](https://github.com/NeOMakinG))
+
+### UX Designer
+
+UX Designers make sure that interfaces follow the best practices in usability and design.
+
+Current UX designers:
+
+- Scott Cosio ([@prestascott](https://github.com/prestascott))
+- Tristan Lehot ([@TristanLDD](https://github.com/TristanLDD))
+
+### Quality Assurance (QA)
+
+Issue managers qualify new issues to avoid duplicates and, if they are indeed defects in the software, their severity.
+
+Current issue managers:
+
+- Florine Hea ([@florine2623](https://github.com/florine2623))
+- Hana Rebai ([@HanaRebaiQA](https://github.com/HanaRebaiQA))
+- Hibatallah Aouadni ([@hibatallahAouadni](https://github.com/hibatallahAouadni))
+- Khouloud Belguith ([@khouloudbelguith](https://github.com/khouloudbelguith))
+- Marwa Chelly ([@marwachelly](https://github.com/marwachelly))
+
+QA analysts make sure that the quality of the software improves over time, by verifying that code changes behave as expected.
+
+Current QA analysts:
+
+- Florine Hea ([@florine2623](https://github.com/florine2623))
+- Hana Rebai ([@HanaRebaiQA](https://github.com/HanaRebaiQA))
+- Hibatallah Aouadni ([@hibatallahAouadni](https://github.com/hibatallahAouadni))
+- Khouloud Belguith ([@khouloudbelguith](https://github.com/khouloudbelguith))
+- Marwa Chelly ([@marwachelly](https://github.com/marwachelly))
+- Robin Fischer ([@Robin-Fischer-PS](https://github.com/Robin-Fischer-PS))
+- Sarah Dib ([@sarahdib](https://github.com/sarahdib))
+
+QA automation engineers write automated functional tests, which help accelerate the work of QA analysts.
+
+Current QA automation engineers:
+
+- Nesrine Abdmouleh ([@nesrineabdmouleh](https://github.com/nesrineabdmouleh))
+- Stéphane Decisy ([@SD1982](https://github.com/SD1982))
+- Boubker Bribri ([@boubkerbribri](https://github.com/boubkerbribri))
+
+### Wording Manager
+
+Wording managers ensure that wordings are adequate, follow the project's tone of voice, and are categorized properly.
```

Comment:
```suggestion
Product Content Managers are responsible for improving the user experience by ensuring the quality and consistency of software content and user documentation.
```

Reply:
I chose "wording" because I didn't think of documentation. Documentation management is a different role that could be done by different contributors. So you're actually doing two roles :P

How about I add a specific role for that?

Reply:
Sounds good ;-)

Reply:
Alright!